test/test-greet.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const expect = require('chai').expect;
+
+let greetings = require('../lib/greet.js');
+
+describe('greetings', function(){
+  it('should exist', function(){
+    expect(greetings).to.not.be.undefined;
+  });
+  it('should return a string', function(){
+    expect(greetings.greet('maria')).to.be.a('string');
+  });
+  it('should concatinate hello with name input', function(){
+    expect(greetings.greet('scott')).to.equal('hello scott');
+  });
+});
+
+const assert = require('assert');
+
```

Comment:
way to dig into the chai docs. this practice will help solidify the concepts we learn, as well as help you grow as a dev. What other data types can you check against?